inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/TDSQLPostgresLoadNode.java

```diff
@@ -0,0 +1,117 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.inlong.sort.protocol.node.load;
+
+import com.google.common.base.Preconditions;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonTypeName;
+import org.apache.inlong.sort.protocol.FieldInfo;
+import org.apache.inlong.sort.protocol.constant.PostgresConstant;
+import org.apache.inlong.sort.protocol.enums.FilterStrategy;
+import org.apache.inlong.sort.protocol.node.LoadNode;
+import org.apache.inlong.sort.protocol.transformation.FieldRelationShip;
+import org.apache.inlong.sort.protocol.transformation.FilterFunction;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * TDSQLPostgres load node can load data into TDSQL Postgres
+ * @see <a herf="https://cloud.tencent.com/product/tbase">TDSQL Postgres</a>
+ * TDSQL Postgres is an enterprise-level distributed HTAP database. Through a single database cluster to provide
+ * users with highly consistent distributed database services and high-performance data warehouse services,
+ * a set of integrated enterprise-level solutions is formed.
+ */
+@EqualsAndHashCode(callSuper = true)
+@JsonTypeName("tdsqlPostgresLoad")
+@Data
+@NoArgsConstructor
+public class TDSQLPostgresLoadNode extends LoadNode implements Serializable {
```

Comment:
It is recommended to use TDSQLLoadNode replace TDSQLPostgresLoadNode.

Reply:
> It is recommended to use TDSQLLoadNode replace TDSQLPostgresLoadNode.

TDSQL include other database. not only Postgres.